app/src/components/shared/RegistrationCustomOption.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import { Checkbox } from "flowbite-svelte";
+
+  export let form;
+  export let option;
+  export let label;
+</script>
+
+<div class="flex flex-col gap-1">
+  <span class="block text-sm font-bold text-gray-900 rtl:text-right dark:text-gray-300"
+    >{label}</span
+  >
+  <Checkbox value={option} name="customOptions" bind:group={$form.customOptions}>Ja</Checkbox>
```

Comment:
Skal vi kun ha mulighet for custom felt som er checkbokser? Tenker vi bør ha inputfelt også, men vi kan jo starte med dette. Ser bra ut da 🌟

Reply:
Det kan være at vi trenger begge, men tenkte at checkboxer var "viktigst" siden dataen blir strukturert 👍